### PR TITLE
52408_Init layout

### PR DESCRIPTION
**Pull request by fonqhoanq**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52408)

## WHAT (optional)
- Init layout
- Set i18n
## HOW
- follow init layout guide
- follow set i18n guide

## WHY (optional)
Screenshot from 2022-08-16 11-24-33

### CODE REVIEW

**Comment by fonqhoanq** [comment]
ready

**Review by diemntn-0165 — APPROVED** [review]

**Comment by fonqhoanq** [comment]
ready

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,24 @@
 class ApplicationController < ActionController::Base
+  before_action :check_role_user
+  before_action :set_locale
```
Comment (manhnt-0960) [inline app/controllers/application_controller.rb]: gộp lại 1 dòng

File: app/views/layouts/application_admin.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= full_title yield :title %></title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_link_tag 'application_admin', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application_admin', 'data-turbolinks-track': 'reload' %>
```
Comment (manhnt-0960) [inline app/views/layouts/application_admin.html.erb]: "

**Comment by fonqhoanq** [comment]
ready

**Pull request merged**